### PR TITLE
[skip-ci] Added S. Ramanujan's Pi Infinite Series tutorial

### PR DESCRIPTION
**Pull request by AdvaitDhingra**
# This Pull request:

This tutorial demonstrates the use of factorials and infinite series on a cool example.

## Changes or fixes:

Added the tutorial file.

## Checklist:

- [x] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

**Review by bellenot — CHANGES_REQUESTED** [review]
Fix indentation (don't use TABs)

**Comment by AdvaitDhingra** [comment]
> Fix indentation (don't use TABs)

Ah, okay.

**Comment by AdvaitDhingra** [comment]
Is that better?

**Comment by bellenot** [comment]
yes, but there are still TABS...

File: tutorials/math/RamanujanPiInfiniteSeries.C [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+void RamanujanPiInfiniteSeries() {
+   Int_t iterations;
+
+   std::cout << "Number of Iterations: " << std::endl;
+   std::cin >> iterations;
+
+   Double_t summation;
+
+   for (int n = 0; n < iterations; n++) {
+	    summation += (TMath::Factorial(4*n) / std::pow(TMath::Factorial(n), 4)) * ((26390*n + 1103) / std::pow(396, 4*n));
```
Comment (bellenot) [inline tutorials/math/RamanujanPiInfiniteSeries.C:24]:
```suggestion
      summation += (TMath::Factorial(4*n) / std::pow(TMath::Factorial(n), 4)) * ((26390*n + 1103) / std::pow(396, 4*n));
```

**Comment by AdvaitDhingra** [comment]
> yes, but there are still TABS...

Ah ok, so I delete the tabs and just do three spaces?

File: tutorials/math/RamanujanPiInfiniteSeries.C [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   }
+
+   Double_t pi = 1/((TMath::Sqrt(8) / 9801) * summation);
+
+	 std::cout << std::setprecision(iterations) << pi << std::endl;
```
Comment (bellenot) [inline tutorials/math/RamanujanPiInfiniteSeries.C:29]:
```suggestion
      std::cout << std::setprecision(iterations) << pi << std::endl;
```

**Review by bellenot — CHANGES_REQUESTED** [review]

File: tutorials/math/RamanujanPiInfiniteSeries.C [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+   Double_t summation;
+
+   for (int n = 0; n < iterations; n++) {
+      summation += (TMath::Factorial(4*n) / std::pow(TMath::Factorial(n), 4)) * ((26390*n + 1103) / std::pow(396, 4*n));
+   }
+
+   Double_t pi = 1/((TMath::Sqrt(8) / 9801) * summation);
+
+      std::cout << std::setprecision(iterations) << pi << std::endl;
```
Comment (bellenot) [inline tutorials/math/RamanujanPiInfiniteSeries.C:29]:
```suggestion
   std::cout << std::setprecision(iterations) << pi << std::endl;
```

**Review by bellenot — COMMENTED** [review]
Sorry, my bad

**Comment by AdvaitDhingra** [comment]
Okay, but isn't the std::cout at the end a bit too much to the right? Or is this what we want?

**Comment by AdvaitDhingra** [comment]
I can squash the commits.

File: tutorials/math/RamanujanPiInfiniteSeries.C (hunk 3 above)
Comment (bellenot) [inline tutorials/math/RamanujanPiInfiniteSeries.C:29]: Still not aligned
```suggestion
   std::cout << std::setprecision(iterations) << pi << std::endl;
```

**Review by bellenot — CHANGES_REQUESTED** [review]

**Comment by AdvaitDhingra** [comment]
I think now the alignment is ok 😂

**Pull request closed without merging**